stripe/stripe_object.py

```diff
@@ -51,17 +84,17 @@ def default(self, obj):
 
     def __init__(
         self,
-        id=None,
-        api_key=None,
-        stripe_version=None,
-        stripe_account=None,
-        last_response=None,
-        **params
+        id: Optional[str] = None,
+        api_key: Optional[str] = None,
+        stripe_version: Optional[str] = None,
+        stripe_account: Optional[str] = None,
+        last_response: Optional[StripeResponse] = None,
+        **params: Any
```

Comment:
_retrieve_params is Dict[str, Any] can we type params the same?

Reply:
I think we should do something here, but you get 300 type errors if you try this, because you have to handle all the unpacked `Params` typedDicts here. I'll add a TODO.

Reply:
TODO added in [Address code review](https://github.com/stripe/stripe-python/pull/1088/commits/067ab1bfd6a74acd7cb4bb8c4f7e878a2d0e4661)